src/omnicore/rpctx.cpp

```diff
@@ -487,6 +489,73 @@ static UniValue omni_senddexaccept(const JSONRPCRequest& request)
     }
 }
 
+#ifdef ENABLE_WALLET
```

Comment:
Mmh in the other calls only some parts are wrapped:

https://github.com/OmniLayer/omnicore/blob/f2660c5ce9a65d3fa0b65005e4291aaca00418e5/src/omnicore/rpctx.cpp#L418

Not sure what would happen, if the wallet is disabled and the RPC is called in this case?

Reply:
@dexX7 None of the calls in this file are added when as per the line below and cannot be called.

https://github.com/OmniLayer/omnicore/blob/938734cab54f81dda84698f498526517026b6832/src/omnicore/rpctx.cpp#L1721

I was planning on taking another look at some point as to whether all calls in this file could wrapped in mass with the ENABLE_WALLET conditional, it would certainly make the code in the calls a bit tidier. I wanted to look at these and all RPC calls that use the wallet.